lib/scout_apm/auto_instrument/rails.rb

```diff
@@ -0,0 +1,73 @@
+
+# In order for this to work, you must add `gem 'parser'` to your Gemfile.
+require 'parser/current'
```

Comment:
Thinking we'll be more defensive here and use conditionals around our definitions and code in order to prevent things blowing up if someone doesn't include `parser` or we're running in an older ruby version, non MRI, etc.

Reply:
Okay I wrapped it and `rescue LoadError`. Let's see how this works on travis.